src/components/content/Content.css

```diff
@@ -0,0 +1,66 @@
+.Content {
+   margin-left: auto;
+   margin-right: auto;
+   background-color: #f0e7e9;
+   padding-top: 150px;
+   padding-bottom: 250px;
+   border-bottom: #c9c9c9 1.5px solid;
+}
+
+.Catchword {
+   text-align: center;
+}
+
+.row {
+   margin-right: 0px;
+}
+
+#P1 {
+   margin-top: 50px;
+   margin-bottom: 5px;
+}
+
+#P2 {
+   margin-bottom: 5px;
+}
+
+.Content2 {
+   margin-left: auto;
+   margin-right: auto;
+   min-height: 500px;
+   border-bottom: #c9c9c9 1px solid;
+}
+
+#Members {
+   text-align: center;
+   margin-top: 50px;
+}
+
+.RowMembers {
```

Comment:
podobnie tutaj: `.row-members`, `#members`, natomiast `.row-members-2` mówi mi trochę, że to jakiś modyfikator elementu `row-members` więc zastanowiłbym się, które linijki się powtarzają i jak to wydzielić do klas wspólnych - każdy nasz element w komponencie może mieć ich kilka więc śmiało :)